$NetBSD: patch-autosetup_autosetup-find-tclsh,v 1.3 2024/11/08 00:47:43 js Exp $

Treat CC_FOR_BUILD as a single command, not a list of separate possible
commands.

--- autosetup/autosetup-find-tclsh.orig	2024-11-06 12:59:09.000000000 +0000
+++ autosetup/autosetup-find-tclsh
@@ -8,7 +8,7 @@ for tclsh in ./jimsh0 $autosetup_tclsh j
     { $tclsh "$d/${1-autosetup-test-tclsh}"; } 2>/dev/null && exit 0
 done
 echo 1>&2 "No installed jimsh or tclsh, building local bootstrap jimsh0"
-for cc in ${CC_FOR_BUILD:-cc} gcc; do
+for cc in "${CC_FOR_BUILD:-cc}" gcc; do
     { $cc -o jimsh0 "$d/jimsh0.c"; } >/dev/null 2>&1 || continue
     ./jimsh0 "$d/${1-autosetup-test-tclsh}" && exit 0
 done
